lms/templates/instructor/edx_ace/enrolledunenroll/email/body.html

```diff
@@ -4,37 +4,47 @@
-    <tr>
-        <td>
-            <h1>
-                {% filter force_escape %}
-                {% blocktrans %}
-                    You have been unenrolled from {{ course_name }}
-                {% endblocktrans %}
-                {% endfilter %}
-            </h1>
 
-            <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}You have been unenrolled from {{ course_name }} at {{ site_name }} by a member of the course staff. This course will no longer appear on your {{ site_name }} dashboard.{% endblocktrans %}
-                {% endfilter %}
-                <br />
-            </p>
+   <tr>
+        <td style="width: 100%;
+            max-width: 800px;
+            background-color: #FFF;
+            color: #000000;
+            margin: auto;
+            text-align: center;
+            border: 1px solid #F3F3F3;">
 
-            <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}Your other courses have not been affected.{% endblocktrans %}
-                {% endfilter %}
-                <br />
-            </p>
+            <img src="{% static 'pearson-pols-theme/images/PearsonLogo_Primary_Blk_RGB.png' %}" style="width: 150px; padding: 5px;" />
 
-            <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}This email was automatically sent from {{ site_name }} to {{ full_name }}{% endblocktrans %}
-                {% endfilter %}
+            <div style="width: 90%;
+                        background-color: #D4EAE4;
+                        text-align: left;
+                        padding: 2% 5%;
+                        background-image: url('{% static 'pearson-pols-theme/images/laptop_transparent-bg.png' %}');
+                        background-size: 50%;
+                        background-position: right bottom;
+                        background-repeat: no-repeat;">
+
+                <h1 style="font-family:'Playfair Display', serif; font-weight: 700;">
+                     {% blocktrans %}You have been unenrolled{% endblocktrans %}
+                </h1>
+                <p style="padding: 0px;
+                          width: 50%;
+                          min-width: 300px;">
+                    {% blocktrans %}You have been unenrolled from {{ course_name }} at {{ site_name }} by a member of the course staff. This course will no longer appear on your {{ site_name }} dashboard. Your other courses have not been affected. {% endblocktrans %}
+                    <br />
+                </p>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <p>
+                 {% blocktrans %}This email was automatically sent from {{ site_name }} to {{ full_name }}{% endblocktrans %}
                 <br />
             </p>
         </td>
     </tr>
+
```

Comment:
It seems like this blank line is not required.

Reply:
There are many details in this template, this template is taken from the existing one, it's not the purpose of this PR to fix them, but to fix the main issue, however, applied.